src/main/java/org/reaktivity/nukleus/maven/plugin/internal/generate/String16FlyweightGenerator.java

```diff
@@ -221,7 +222,7 @@ private MethodSpec toStringMethod()
     private MethodSpec length0Method()
     {
         return methodBuilder("length0")
-                .addModifiers(PRIVATE)
+                .addModifiers(PUBLIC)
```

Comment:
We named this method `length0()` because it is internal, let's review the need for a change to `public` access modifier and if still required then we should consider renaming the method.